packages/react-devtools-shared/src/__tests__/console-test.js

```diff
@@ -279,7 +279,8 @@ describe('console', () => {
     expect(mockWarn.mock.calls[0][0]).toBe('active warn');
     expect(normalizeCodeLocInfo(mockWarn.mock.calls[0][1])).toEqual(
       supportsOwnerStacks
-        ? '\n    in Parent (at **)'
+        ? // TODO: It would be nice to have a Child stack frame here since it's just the effect function.
```

Comment:
This only has the `useEffect(() => ` inner function on the stack trace which don't currently have the `react-bottom-stack-frame` which means they're filtered out since we don't know which frames to include.

Even if we did it wouldn't have a name. It might be nice to automatically name the effect functions based on the component/hook that they're part of. Similar to how class methods are named based on both the class instance and the method.